drape_frontend/apply_feature_functors.cpp

```diff
@@ -1052,15 +1003,6 @@ void ApplyLineFeatureAdditional::GetRoadShieldsViewParams(ref_ptr<dp::TextureMan
     titleDecl.m_secondaryTextFont.m_size *= 0.9f;
     titleDecl.m_secondaryOffset = m2::PointD(0.0f, 3.0 * mainScale);
   }
-
-  // Special priority for road shields in navigation style.
-  if (GetStyleReader().IsCarNavigationStyle())
-  {
-    textParams.m_specialDisplacement = poiParams.m_specialDisplacement
-      = symbolParams.m_specialDisplacement = SpecialDisplacement::SpecialMode;
-    textParams.m_specialPriority = poiParams.m_specialPriority
-      = symbolParams.m_specialPriority = CalculateNavigationRoadShieldPriority();
-  }
```

Comment:
We can raise the shield's priority in Navigation mode by editing the correspondent style, right?
In case we want to not displace shields at all for example ..

Reply:
Yeap!